### PR TITLE
Update 10-Timer.md

### LINKED ISSUE #86: 10-Timer.md Bonus#1:
The idea given in bonus example 1 of lab 10/timers lab was to update the player's current score until they have lost. It was discussed that it can be updated in actionPerformed and to add checks to see if the player has lost. Reading the explanation is a little confusing. An idea to add is to create snippets of sample code/examples for the person doing the lab to see. 
2-3 examples can be provided:
1.a) How to update/code string in actionPerformed that displays score.
1.b) How to count player score/kills. It can be an extra sentence in the instructions that tell the person doing the lab that an integer can be updated near the function used to remove enemies from the screen upon collision.
2) What function to use to check player loss, but leave where to place code to the person doing the lab.

### PR DESCRIPTION
**Pull request by rthao1**
fixes #86

### CODE REVIEW

File: docs/labs/10-Timer.md [hunk 1]
```diff
@@ -609,10 +609,35 @@ For example,
 you could add a label that is continually updated or set in `actionPerformed`
 that would update the number of green squares that you have removed from the screen
 (see the version on the left).
+
+```java
```
Comment (onzfonz) [inline docs/labs/10-Timer.md:613]: please provide some intro to this code snippet.

**Review by onzfonz — CHANGES_REQUESTED** [review]
once you have this, we'll do one more round of changes at least that will be related to me suggesting changes

**Comment by rthao1** [comment]
fixes #86 pt2, I added some intro to the code

File: docs/labs/10-Timer.md (hunk 1 above)
Comment (rthao1) [reply docs/labs/10-Timer.md:616]: I added some intro to the code in the latest pull request

File: docs/labs/10-Timer.md [hunk 2]
```diff
@@ -609,10 +609,38 @@ For example,
 you could add a label that is continually updated or set in `actionPerformed`
 that would update the number of green squares that you have removed from the screen
 (see the version on the left).
+Below is a snippet of code that will act as a way to update the player's score. You will need a
+`GLabel` to display the string on the game window. The basic functions of the code will
+continuously remove and add the players score in real time.
+
+```java
+public void actionPerformed(ActionEvent e) {
+    ...
```
Comment (onzfonz) [inline docs/labs/10-Timer.md:618]:
```suggestion
    // ...
```

File: docs/labs/10-Timer.md [hunk 3]
```diff
@@ -609,10 +609,38 @@ For example,
...
+    remove(displayScore);
+    displayScore = new GLabel("Score: " + score,scoreX,scoreY);
+    add(displayScore);
+    ...
```
Comment (onzfonz) [inline docs/labs/10-Timer.md:622]:
```suggestion
    // ...
```

File: docs/labs/10-Timer.md [hunk 4]
```diff
@@ -609,10 +609,38 @@ For example,
...
+}
+```
+
+If you are struggling tracking the players score or updating the label,
+try declaring a global variable that is updated when the enemy is killed (hint: when are they removed?).
```
Comment (onzfonz) [inline docs/labs/10-Timer.md:627]:
```suggestion
try declaring a global variable that is updated when the enemy is killed. 

> hint: *when are they removed?*

```

File: docs/labs/10-Timer.md [hunk 5]
```diff
@@ -609,10 +609,38 @@ For example,
...
+Remember that the main idea for displaying the score is to constantly display the score being tracked.
 Or you could instead (or in addition)
 add in checks to say if there are more than `MAX_ENEMIES` on the screen,
 then stop the timer,
-remove everything and add a message telling them they lost.
+remove everything and add a message telling them they lost. Here's what an example for losing looks like.
```
Comment (onzfonz) [inline docs/labs/10-Timer.md:632]:
```suggestion
remove everything and add a message telling them they lost.
Here's what an example for losing looks like.
```

File: docs/labs/10-Timer.md [hunk 6]
```diff
@@ -609,10 +609,38 @@ For example,
...
+However, it will be up to you to choose where to add this code.
+
+```java
+if (enemies.size() == MAX_ENEMIES) {
+    GLabel lose = new GLabel("lose message",xLoc,yLoc);
```
Comment (onzfonz) [inline docs/labs/10-Timer.md:637]:
```suggestion
    GLabel lose = new GLabel("you lose",xLoc,yLoc);
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
Please use the add batch suggestion and then re request review.
Also make sure you're branch is updated

**Comment by onzfonz** [comment]
You'll need to fix the remaining errors @rthao1

File: docs/labs/10-Timer.md [hunk 7]
```diff
@@ -609,10 +609,42 @@ For example,
 you could add a label that is continually updated or set in `actionPerformed`
 that would update the number of green squares that you have removed from the screen
 (see the version on the left).
+Below is a snippet of code that will act as a way to update the player's score. You will need a
+`GLabel` to display the string on the game window. The basic functions of the code will
```
Comment (onzfonz) [inline docs/labs/10-Timer.md:613]:
```suggestion
`GLabel` to display the string on the game window.
The basic functions of the code will
```

File: docs/labs/10-Timer.md [hunk 8]
```diff
@@ -609,10 +609,42 @@ For example,
...
+
+> hint: *when are they removed?*
+
+Remember that the main idea for displaying the score is to constantly display the score being tracked.
 Or you could instead (or in addition)
 add in checks to say if there are more than `MAX_ENEMIES` on the screen,
 then stop the timer,
 remove everything and add a message telling them they lost.
+Here's what an example for losing looks like.
+However, it will be up to you to choose where to add this code.
```
Comment (onzfonz) [inline docs/labs/10-Timer.md:637]:
```suggestion
However, *it will be up to you to choose where to add this code.*
```

**Review by onzfonz — CHANGES_REQUESTED** [review]
Should be the last set of changes. Makes sure branch is updated and tests pass

**Review by rthao1 — COMMENTED** [review]
I believe the branch should be up-to-date and all tests are passed

**Review by onzfonz — APPROVED** [review]
LGTM

**Comment by onzfonz** [comment]
@rthao1 all you need to do is cleen the green merge pull request button to merge!  If your branch becomes outdated, you'll need to do that first

**Pull request merged**